src/plugins/tables/_base.scss

```diff
@@ -189,6 +189,18 @@ table {
 				}
 			}
 
+			button {
+				&.sorting-cnt {
+					-moz-appearance: none;
+					-webkit-appearance: none;
```

Comment:
Usually the prefixed CSS are already managed by PostCSS during the build. Usually here we are using the naming from the W3C standard like "appearance".